src/rpc/rawtransaction.cpp

```diff
@@ -211,7 +211,7 @@ UniValue gettxoutproof(const UniValue& params, bool fHelp)
             "\nNOTE: By default this function only works sometimes. This is when there is an\n"
             "unspent output in the utxo for this transaction. To make it always work,\n"
             "you need to maintain a transaction index, using the -txindex command line option or\n"
-            "specify the block in which the transaction is included in manually (by blockhash).\n"
+            "specify the block in which the transaction is included manually (by blockhash).\n"
```

Comment:
Agreed. Overall the entire NOTE is painful for me to read.

Saying thing like "By default this function only works sometimes" and "To make it always work" is pretty strange. There must be a better way to express this note than flail about like that.